doc/developer.md

```diff
@@ -1,17 +1,35 @@
 \page develmanual Developer manual
 [TOC]
 
- * General topics
-   - \subpage develWithRadium
-   - \subpage develCodingConvention
-   - \subpage develoldmanual
- * Main modules of the API
-   - \subpage develbuildchain
+- \subpage develWithRadium
+- \subpage develCodingConvention
+- \subpage develbuildchain
+- \subpage api
+- \subpage plugin
+- \subpage apps
+- \subpage develoldmanual
+
+\page api API
+- \subpage core
+- \subpage engine
+- \subpage gui
+
+\page core Core
    - \subpage develmeshes
+   
+\page engine
```

Comment:
```suggestion
\page Engine
```

Reply:
acutally this is the page tag, like develmeshes, we have used only downcase name for that.
Page title has to be added.